app/models/comment.rb

```diff
@@ -20,6 +20,10 @@ def self.comments_to_user_activity(user_id)
   end
 
   def find_user
-    user.name unless user.name.nil?
+    return user.name unless user.nil?
+  end
+
+  def update_date
```

Comment:
I really like that you extracting this logic back to the model so our views are cleaner.